pyfolio/risk.py

```diff
@@ -432,9 +432,22 @@ def compute_volume_exposures(shares_held, volumes, percentile):
     shorted_frac = shares_shorted.divide(volumes)
     grossed_frac = shares_grossed.divide(volumes)
 
-    longed_threshold = 100*longed_frac.quantile(percentile, axis='columns')
-    shorted_threshold = 100*shorted_frac.quantile(percentile, axis='columns')
-    grossed_threshold = 100*grossed_frac.quantile(percentile, axis='columns')
+    # NOTE: To work around a bug with nan-handling in pandas 0.18,
+    #       drop all-nan rows before the call to `quantile`, and then
+    #       restore them by reindexing the final result. This is fixed
+    #       in pandas 0.19.
+    longed_threshold = 100*longed_frac.dropna(how='all').quantile(
```

Comment:
will this fail flake8 if there aren't spaces around `*`?

Reply:
Apparently not, as I was just leaving this as it was. For some reason thought we were consistently not including the whitespace in this project, but now that I look again it seems like we do include it more often that not. I'll update this and the other occurrences in this file.